server/db/index.js

```diff
@@ -0,0 +1,13 @@
+const { Client } = require('pg');
+
+const client = new Client({
```

Comment:
I am not familiar with Client, but this looks so clean

Reply:
❤️